_episodes/02-shell_to_cwl.md

```diff
@@ -153,9 +153,9 @@ The RNA-seq data from the introduction episode will be used for the first CWL wo
 The first step of RNA-sequencing analysis is a quality control of the RNA reads using the `fastqc` tool.
 This tool is already available to use so there is no need to write a new CWL tool description.
 
-This is the workflow file (`rna_seq_workflow.cwl`).
+This is the workflow file (`rna_seq_workflow1.cwl`).
```

Comment:
This suggestion is probably overkill, so feel free to shoot it down - but should we mark all files with the episode they are used in? If we do my preferred style for this would be `ep2_rna_seq_workflow.cwl` (putting the episode ID at the start of the filename will ensure all files for a particular episode appear alongside each other).